{
  "Header": {
    "Original": "CVE-2018-16048 (An issue was discovered in GitLab Community and Enterprise Edition bef ...)",
    "Line": 330328,
    "ID": "CVE-2018-16048",
    "Description": "(An issue was discovered in GitLab Community and Enterprise Edition bef ...)"
  },
  "Annotations": [
    {
      "Original": "- gitlab \u003cnot-affected\u003e (Only affects Enterprise edition)",
      "Line": 330329,
      "Type": "package",
      "Package": "gitlab",
      "Kind": "not-affected",
      "Description": "Only affects Enterprise edition"
    },
    {
      "Original": "NOTE: https://gitlab.com/gitlab-org/gitlab-ce/issues/49947",
      "Line": 330330,
      "Type": "NOTE",
      "Description": "https://gitlab.com/gitlab-org/gitlab-ce/issues/49947"
    },
    {
      "Original": "NOTE: https://about.gitlab.com/2018/08/28/security-release-gitlab-11-dot-2-dot-2-released/",
      "Line": 330331,
      "Type": "NOTE",
      "Description": "https://about.gitlab.com/2018/08/28/security-release-gitlab-11-dot-2-dot-2-released/"
    }
  ]
}